de.willuhn.Jameica.appdata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>de.willuhn.Jameica</id>
+  <name>Jameica</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0+</project_license>
+  <developer_name>Olaf Willuhn</developer_name>
+  <summary>Application-Platform similar to OSGI</summary>
+  <description><p>Jameica is an Application-Platform written in Java containing a SWT-UI. It provides different services (GUI-Toolkit, Logging, Security, Backup, Lifecycle-Management, Message-Bus) to the installed plugins. It's a kind of runtime environment similar to OSGi.</p></description>
```

Comment:
Given that this app is only available in german it'd be good to have localized summary and descriptions in that language.

Reply:
Good point - I've updated the app metadata.